usecases/auth/authentication/apikey/db_users.go

```diff
@@ -213,6 +243,17 @@ func (c *DBUser) CheckUserIdentifierExists(userIdentifier string) (bool, error)
 	return ok, nil
 }
 
+func (c *DBUser) UpdateLastUsedTimeStamp(users map[string]time.Time) {
```

Comment:
```suggestion
func (c *DBUser) UpdateLastUsedTimestamp(users map[string]time.Time) {
```
nit: one word (like [in protobuf](https://pkg.go.dev/github.com/golang/protobuf/ptypes/timestamp#Timestamp), for example)